bin/install

```diff
@@ -5,7 +5,15 @@ install_postgres_app() {
   local version=$2
   local install_path=$3
 
-  ln -s /Applications/Postgres.app/Contents/Versions/$version/bin $install_path
+  local version_path=/Applications/Postgres.app/Contents/Versions/$version/bin
+
+  if [ -d $version_path ]; then
+    ln -s $version_path $install_path
+    echo "Installed version $version"
+  else
+    echo -e "Version $version is not supported by Postgres.app.\nUse 'asdf postgres-app list-all' to see all supported versions."
```

Comment:
maybe the message should be:
> not supported by your installed Postgres.app

We could also give a hint that usually, binaries with the desired version can be downloaded.

Reply:
Good point 👍🏻

Reply:
How about:

```
Version $version is not supported by your installed Postgres.app.
Use 'asdf list-all postgres-app' to see all supported versions of your installed Postgres.app.
If the desired version is missing make sure to check https://postgresapp.com/downloads.html for new versions of Postgres.app.
```